app/services/folio/courses.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Folio
+  class Courses
+    class << self
+      delegate :courses, to: :instance
+    end
+
+    def self.instance
+      @instance ||= new
+    end
+
+    attr_reader :folio_client
+
+    def initialize(folio_client: FolioClient.new)
+      @folio_client = folio_client
+    end
+
+    # Write courses from FOLIO to database
+    def sync_courses!
```

Comment:
Is there a mechanism for removing old course reserves data?

Reply:
You and @corylown below have the same question so will just answer here.  I had not put that in and should. I could empty out the whole database and rewrite to it (once I know for sure I do have any data coming back) instead of handling atomic deletes.  Or, if we would rather do atomic deletes, find which ones did not come through in the latest list and remove them.

Reply:
I'd lean toward atomic deletes to avoid moments throughout the day when the courses table would be empty.

Reply:
Ok. Will look into that. Thank you!

Reply:
Added deletion step.